common/stringFunctions.lua

```diff
@@ -74,8 +69,5 @@ if not string.formatTime then
 		end
 	end
 end
--- Unit test:
--- print(string.partition("blaksjdfsaldkj","ldkj"))
--- print(string.partition("blaksjdfsaldkj","aks"))
-
 
+return string
```

Comment:
This return is not required, as the `string` table is is being mutated directly.